test/moonwall.config.json

```
@@ -1238,6 +1238,23 @@
                     "endpoints": ["ws://127.0.0.1:9950"]
                 }
             ]
+        },
+        {
+            "name": "dancebox_smoke_moonbase",
```

```suggestion
            "name": "dancebox_smoke_moonbase_test",
```

since its only for testing purposes